migrations/sql-annex/V2018.02.15.1500__RecreateBurdenEstimateStochasticWithSmallIntFields.sql

```diff
@@ -0,0 +1,24 @@
+DROP TABLE burden_estimate_stochastic;
+
+CREATE TABLE burden_estimate_stochastic (
+  id BIGSERIAL NOT NULL,
+  burden_estimate_set integer NOT NULL,
+  country text NOT NULL,
+  country_nid SMALLINT NULL,
+  year SMALLINT NOT NULL,
+  burden_outcome SMALLINT NOT NULL,
+  stochastic boolean NOT NULL,
+  value numeric,
```

Comment:
@MartinEden and I had a big back and forth on this and tentatively decided that `DECIMAL(15)` might be the way to go (same as `NUMERIC(15)` and same precision as a double).  As is an unqualified `DECIMAL` / `NUMERIC` type uses almost unbounded space ("up to 131072 digits before the decimal point; up to 16383 digits after the decimal point").

However, reading [the docs](https://www.postgresql.org/docs/10/static/datatype-numeric.html) further I see

> However, calculations on numeric values are very slow compared to the integer types, or to the floating-point types described in the next section

So I am going to suggest we give up and go with floating point numbers :)

I guess if this is going to be merged in before Martin is back we should leave as you have it and open a new tickt.  Otherwise we might deal with it at the same time